### DISCUSSION TITLE
Amazon Approved - How to activate Developer App in 'Ready for Publishing'?

### DISCUSSION THREAD

**Discussion by Tanveer-LowCoder** [amzn/selling-partner-api-models#3135] [General]
![image](https://user-images.githubusercontent.com/76622434/132118513-badb8be8-b39c-43da-8a46-549b69f370be.png)

Team, an App that I built for Amazon Developer using the SP API has now been approved and this is the status that is shown. It's been like this for the last 3 months with no further update from Amazon MWS or any other teams (and support has been unhelpful). Can you recommend what was the next step that you did?

**Comment by ezpinsky** [comment]
Contact mws developer support [here](https://sellercentral.amazon.com/gp/mws/contactus.html) and give them your app ID. Tell them the situation. If they don't respond in a day or two then reply to the same case asking them for a timely update.

Best of luck!
